README.md

```diff
@@ -2,9 +2,11 @@
 
 Serves as a visual Web Browser for the [`package-backend`](https://github.com/pulsar-edit/package-backend).
 
-Currently this matches very closely with the design of the original package viewer from upstream Atom.
-
-Although does fully support multiple themes being added on later with a theme switcher.
+While the site is still in it's early stages it has an original design and supports several themes already:
+  * Atom.io Theme - Theme Based off the Original Atom Package Browser
+  * GitHub Dark - A theme Based off GitHub Dark Theme
+  * Dracula - <@kyjus25 Please feel free to write an accurate description>
```

Comment:
"Theme based off the versatile [Dracula](https://draculatheme.com/) theme 🧛"

Reply:
Love it, I'll add it when I'm back at home, thanks!